UIKitLesson1/PlayerVC.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  PlayerVC.swift
+//  UIKitLesson1
+//
+//  Created by Kirill Dev on 21.12.2022.
+//
+
+import UIKit
+import AVKit
+
+final class PlayerVC: UIViewController {
+    
+    public var position: Int       = 0
+    public var songs: [Song]       = []
+
+    private var player             = AVAudioPlayer()
+    private let hideButton         = UIButton()
+    private let shareButton        = UIButton()
+    private let trackImageView     = UIImageView()
+    private let currentTimeLabel   = UILabel()
+    private let durationLabel      = UILabel()
+    private let albumTitleLabel    = UILabel()
+    private let trackTitleLabel    = UILabel()
+    private let authorTitleLabel   = UILabel()
+    private let previosTrackButton = UIButton()
+    private let playPauseButton    = UIButton()
+    private let nextTrackButton    = UIButton()
+    
+    private let currentTimeSlider: UISlider = {
+        let slider = UISlider()
+        slider.isContinuous = true
+        slider.tintColor = .green
+        slider.minimumValue = 0.0
+        slider.maximumValue = 1.0
+        return slider
+    }()
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        player.stop()
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupPlayer()
+    }
+    
+    private func setupUI() {
+        [trackImageView, hideButton, shareButton, albumTitleLabel, trackTitleLabel, authorTitleLabel, currentTimeSlider, previosTrackButton, playPauseButton, nextTrackButton, currentTimeLabel, durationLabel].forEach{ view.addSubview($0) }
+        
+        [trackImageView, hideButton, shareButton, albumTitleLabel, trackTitleLabel, authorTitleLabel, currentTimeSlider, previosTrackButton, playPauseButton, nextTrackButton, currentTimeLabel, durationLabel].forEach{ $0.translatesAutoresizingMaskIntoConstraints = false }
```

Comment:
```suggestion
        [trackImageView, hideButton, shareButton, albumTitleLabel, trackTitleLabel, authorTitleLabel, currentTimeSlider, previosTrackButton, playPauseButton, nextTrackButton, currentTimeLabel, durationLabel].forEach{ view.addSubview($0)
        $0.translatesAutoresizingMaskIntoConstraints = false }
```

а почему это нельзя объеденить?)